nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/BulkStatus.java

```diff
@@ -106,38 +112,38 @@ public void setState(State state) {
     }
 
     /**
-     * Gets bulk operation creation date. This corresponds to the bulk command submission date.
+     * Gets bulk action creation instant. This corresponds to the bulk command submission instant.
      *
-     * @return the creation date
+     * @return the creation instant
      */
-    public ZonedDateTime getCreationDate() {
-        return creationDate;
+    public Instant getCreationInstant() {
+        return creationInstant;
     }
 
     /**
-     * Sets bulk operation creation date.
+     * Sets bulk action creation date.
      *
-     * @param creationDate the creation date
+     * @param creationInstant the creation date
      */
-    public void setCreationDate(ZonedDateTime creationDate) {
-        this.creationDate = creationDate;
+    public void setCreationInstant(Instant creationInstant) {
+        this.creationInstant = creationInstant;
     }
 
     /**
-     * Gets number of scrolled document
+     * Gets number of element
      *
-     * @return the number of scrolled document
+     * @return the number of element
```

Comment:
the number of elements touched by the operation or processed ?

Reply:
It's the number touched by the operation/action.
Processed metrics will be done with [NXP-25099](https://jira.nuxeo.com/browse/NXP-25099)